synapse/handlers/room.py

```diff
@@ -1135,7 +1115,13 @@ async def send(
         )
 
         logger.debug("Sending %s in new room", EventTypes.Member)
-        await send(creation_event, creation_context, creator)
+        ev = await self.event_creation_handler.handle_new_client_event(
+            requester=creator,
+            events_and_context=[(creation_event, creation_context)],
+            ratelimit=False,
+            ignore_shadow_ban=True,
+        )
+        last_sent_event_id = ev.event_id
 
         # Room create event must exist at this point
         assert last_sent_event_id is not None
```

Comment:
I think we can remove this assert and the definition of `last_sent_event_id` up on line 1069. It will now always be a `str` and is initially defined on this line.

(I think this is my last comment!)

Reply:
Right thanks for the feedback!